packages/editable/src/editable.tsx

```diff
@@ -160,17 +161,44 @@ if (__DEV__) {
   EditableInput.displayName = "EditableInput"
 }
 
+export interface EditableTextareaProps extends HTMLChakraProps<"textarea"> {}
+
+/**
+ * EditableTextarea
+ *
+ * The textarea used in the `edit` mode
+ */
+export const EditableTextarea = forwardRef<EditableTextareaProps, "textarea">(
+  (props, ref) => {
+    const { getTextareaProps } = useEditableContext()
+    const styles = useStyleConfig("Textarea", props)
```

Comment:
This differs from the implementation of EditableInput. Editable is a MultiPartStyle component and we are using the key `input` there.
Without this change there would be no way to style the EditableTextarea independently of the Textarea component.

Kindly use the useStyles hook and access the `textArea` property. We may have to add a default styling in the theme.

Reply:
Thanks for kind review @TimKolberger !